code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -297,7 +297,7 @@
 	add_fingerprint(usr)
 
 /obj/machinery/atmospherics/unary/cryo_cell/attackby(obj/item/G, mob/user, params)
-	if(istype(G, /obj/item/reagent_containers/glass))
+	if(istype(G, /obj/item/reagent_containers/glass) && !(user.a_intent == INTENT_HARM))
```

Comment:
Could you change all of these checks to use `!=` instead? You can then do away with the new set of parentheses as well.

```suggestion
	if(istype(G, /obj/item/reagent_containers/glass) && user.a_intent != INTENT_HARM)
```

Reply:
Thank you! I've changed it!